src/elements/Avatar.tsx

```diff
@@ -9,6 +9,8 @@ import { space, SpaceProps } from "styled-system"
 
 const sizeValue = ({ size = "" }) => {
   switch (size) {
+    case "xsmall":
```

Comment:
I talked to Brian about the wording on this. He said he'd prefer to keep it to two letter characters (xs, md, lg) but this is good for now. I'll create an issue to update the sizing in a separate ticket.

Reply:
Updating with the next pr since I forgot to include the `xs` value as a valid value for interface attribute.

I believe reaction is the only consumer of this for now that will have to change with this update, right?